test/intl402/DurationFormat/prototype/format/length.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.datetimeformat.prototype.format
+description: >
+  Intl.DateTimeFormat.prototype.format.length is 1.
+info: |
+  Intl.DurationFormat.prototype.format ( duration )
+
+  17 ECMAScript Standard Built-in Objects:
+
+    Every built-in function object, including constructors, has a length
+    property whose value is an integer. Unless otherwise specified, this
+    value is equal to the largest number of named arguments shown in the
+    subclause headings for the function description. Optional parameters
+    (which are indicated with brackets: [ ]) or rest parameters (which
+    are shown using the form «...name») are not included in the default
+    argument count.
+    Unless otherwise specified, the length property of a built-in function
+    object has the attributes { [[Writable]]: false, [[Enumerable]]: false,
+    [[Configurable]]: true }.
+
+includes: [propertyHelper.js]
```

Comment:
This test is missing `features: [Intl.DurationFormat]`.

Reply:
This PR fixes that https://github.com/tc39/test262/pull/3385 , thanks for the comment

Reply:
Thanks a lot for a very fast response and fix.